packages/node_modules/@webex/plugin-meetings/test/integration/spec/testUtils.js

```diff
@@ -134,11 +134,22 @@ const waitForEvents = (scopeEvents) => {
               }
             }
 
-            if (result.length === scopeEvents.length) {
+            if (obj.match) {
+              if (obj.match(value)) {
```

Comment:
```suggestion
            if (obj?.match(value)) {
```

Reply:
I like it. Forgot I could do the `?.` now